
builtin_interfaces/Time stamp
float32 attitude          # attitude in radians

# TOPICS offboard/controller_input_attitude 
